src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6468,25 +6481,37 @@ s32 func_8083F524(PlayState* play, Player* this) {
     return func_8083F360(play, this, 26.0f, this->ageProperties->unk_38 + 5.0f, 30.0f, 0.0f);
 }
 
-s32 func_8083F570(Player* this, PlayState* play) {
-    s16 temp;
+/**
+ * Two exit walls are placed at each end of the crawlspace, separate to the two entrance walls used to enter the
+ * crawlspace. These front and back exit walls are slightly more interior in the crawlspace relative to the front and
```

Comment:
"slightly more interior" doesnt read right to me, though maybe its fine? Id say slightly more inside

Reply:
Or just "further into".